e2e/test/iothub/service/RegistryE2ETests.cs

```diff
@@ -24,6 +25,9 @@ public class RegistryE2ETests : E2EMsTestBase
     {
         private readonly string _idPrefix = $"{nameof(RegistryE2ETests)}_";
 
+        private static readonly IRetryPolicy s_provisioningServiceRetryPolicy = new ProvisioningServiceRetryPolicy();
+        private static readonly HashSet<Type> s_retryableExceptions = new HashSet<Type> { typeof(IotHubServiceException) };
```

Comment:
This concept will need to be revamped now that exception type isn't the deciding factor, rather the ErrorCode or even the IsTransient property.

Reply:
Agreed